packages/core/src/utils/utils.ts

```diff
@@ -148,16 +148,16 @@ export const getBreakpointSize = (array, length) => {
  * @param sizes - An object of size styles
  *
  */
-export const applySizes =
-  (sizes = null, defaultSize = 'medium', mediaQueriesOptions = mediaQueries) =>
-  ({ size }) => {
-    if (sizes && typeof size === 'string') {
-      // prettier-ignore
-      return css`${sizes[size] || sizes[defaultSize] || ''}`
-    }
-    if (sizes && Array.isArray(size)) {
-      // prettier-ignore
-      return css`${sizes[getBreakpointSize(size, 1)]};
+export const applySizes = (sizes = null, defaultSize = 'medium', mediaQueriesOptions = mediaQueries) => ({
```

Comment:
Lots of unexpected formatting changes here as well. We should figure out what's causing this.